src/librbd/ImageWatcher.cc

```diff
@@ -896,6 +896,8 @@ bool ImageWatcher::handle_payload(const FlattenPayload &payload,
 
 bool ImageWatcher::handle_payload(const ResizePayload &payload,
 				  C_NotifyAck *ack_ctx) {
+  return ImageWatcher::handle_payload(UnknownPayload(), ack_ctx);
```

Comment:
Yes, just for testing: make remote return -EOPNOTSUPP. It is not going to be committed.